### PR TITLE
Added gesture for rescheduling

### LINKED ISSUE #14179: Request to make "Reschedule" as one of the command mapping controller button options
Is it possible to make "Reschedule" as one of the command mapping controller button options? I would like to be able to reschedule a card to a different date as needed with a touch of a gesture or button on a game controller without having to tap multiple times to reach the "Reschedule for review in x days"  option for a specific card. 
     Simply put, I would like to be able to do a certain gesture or click a single button to reach the "Reschedule" option like we can do for "Bury card", "Undo", etc. Please let me know if there's already a way to do this! Thank you all so much!!

### PR DESCRIPTION
**Pull request by parthn2**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Fixes #14179 

## Approach
_How does this change address the problem?_

## How Has This Been Tested?
Tested on Android Studio Emulator

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by lukstbit** [comment]
Your changes are making some of the tests to fail. You have to also fix those tests.

File: AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/PreferenceUpgradeService.kt [hunk 1]
```diff
@@ -271,7 +271,8 @@ object PreferenceUpgradeService {
                 Pair(45, ViewerCommand.CHANGE_WHITEBOARD_PEN_COLOR),
                 Pair(41, ViewerCommand.SHOW_HINT),
                 Pair(42, ViewerCommand.SHOW_ALL_HINTS),
-                Pair(43, ViewerCommand.ADD_NOTE)
+                Pair(43, ViewerCommand.ADD_NOTE),
+                Pair(44, ViewerCommand.RESCHEDULE_NOTE)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/PreferenceUpgradeService.kt:275]: not necessary

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/cardviewer/ViewerCommand.kt [hunk 2]
```diff
@@ -71,7 +71,8 @@ enum class ViewerCommand(val resourceId: Int) {
     CHANGE_WHITEBOARD_PEN_COLOR(R.string.title_whiteboard_editor),
     SHOW_HINT(R.string.gesture_show_hint),
     SHOW_ALL_HINTS(R.string.gesture_show_all_hints),
-    ADD_NOTE(R.string.menu_add_note);
+    ADD_NOTE(R.string.menu_add_note),
+    RESCHEDULE_NOTE(R.string.reschedule_card_dialog_message);
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/cardviewer/ViewerCommand.kt:75]: Just `Reschedule` reads better here
```suggestion
    RESCHEDULE_NOTE(R.string.card_editor_reschedule_card);
```

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

**Review by BrayanDSO — APPROVED** [review]
LGTM. Thanks for the contribution

**Review by lukstbit — APPROVED** [review]
Thanks you for contributing!

**Pull request merged**